latest Webpack-cli package making build fail

With the later version of the webpack-cli 4.3.0 the docker build is failing 

```
[webpack-cli] TypeError: compiler.getInfrastructureLogger is not a function
    at CLIPlugin.apply (/usr/src/app/node_modules/webpack-cli/lib/plugins/CLIPlugin.js:80:32)
    at webpack (/usr/src/app/node_modules/webpack/lib/webpack.js:47:13)
    at WebpackCLI.createCompiler (/usr/src/app/node_modules/webpack-cli/lib/webpack-cli.js:1165:24)
    at async WebpackCLI.bundleCommand (/usr/src/app/node_modules/webpack-cli/lib/webpack-cli.js:1292:20)
    at async Command.<anonymous> (/usr/src/app/node_modules/webpack-cli/lib/webpack-cli.js:322:21)
    at async Promise.all (index 1)
    at async Command.<anonymous> (/usr/src/app/node_modules/webpack-cli/lib/webpack-cli.js:708:13)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! my-learning-analytics@ prod: `BABEL_ENV=production webpack --config webpack.config.js --mode production`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the my-learning-analytics@ prod script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-01-05T16_21_41_509Z-debug.log
```

Currently the fix is to label it o 4.2.0 and later deal with the detailed root cause for this

--- a/package.json
+++ b/package.json
@@ -114,6 +114,6 @@
     "url-loader": "1.1.2",
     "webpack": "~4.29.6",
     "webpack-bundle-tracker": "^0.4.3",
-    "webpack-cli": "^4.2.0"
+    "webpack-cli": "4.2.0"
   }
 }

